src/antsibull_docs/docs_parsing/ansible_doc.py

```diff
@@ -64,38 +64,37 @@ def parse_ansible_galaxy_collection_list(json_output: t.Mapping[str, t.Any],
     return result
 
 
-def _call_ansible_version(
+async def _call_ansible_version(
     venv: t.Union['VenvRunner', 'FakeVenvRunner'],
-    env: t.Dict[str, str],
+    env: t.Optional[t.Dict[str, str]],
 ) -> str:
-    venv_ansible = venv.get_command('ansible')
-    ansible_version_cmd = venv_ansible('--version', _env=env)
-    return ansible_version_cmd.stdout.decode('utf-8', errors='surrogateescape')
+    p = await venv.async_log_run(['ansible', '--version'], env=env)
+    return p.stdout
 
 
-def _call_ansible_galaxy_collection_list(
+async def _call_ansible_galaxy_collection_list(
     venv: t.Union['VenvRunner', 'FakeVenvRunner'],
     env: t.Dict[str, str],
```

Comment:
The only one that accepts `None` is `_call_ansible_version()`. Changing all to `t.Optional[]` would simply make the tests code more complicated (since then all three mock functions must handle `None`, and not just one of them), but have no other advantage from my POV.